### ISSUE TITLE
Add way to call ccrba from RobotWrapper

### ISSUE THREAD

**Issue by jviereck** [stack-of-tasks/pinocchio#1110]
In Pinocchio 2.2.2, calling

```
robot.centroidalMomentum(q, dq)
```

fills the `robot.data.Ag` entries. Having version 2.3.1 installed it's necessary to call

```
se3.ccrba(self.robot.model, self.robot.data, q, dq)
```

It would be great if there would be a call like `robot.computeCentroidalMomentumMatrix(q, dq)` or similar on the RobotWrapper.

**Comment by jcarpent** [comment]
Yes, sure. Then the API could be simply:
`robot.computeCentroidalMomentumMatrix(q)`
and we may also add something like:
`robot.computeCentroidalData(q,v)` or maybe a better would be appreciate here.

**Comment by jcarpent** [comment]
Yes, sure. Then the API could be simply:
`robot.computeCentroidalMomentumMatrix(q)`
and we may also add something like:
`robot.computeCentroidalData(q,v)` or maybe a better would be appreciate here.

**Comment by gabrielebndn** [comment]
Why not simply `robot.ccrba`? I don't see the point of having a name which does not resemble at all the one of the normal API

**Comment by jviereck** [comment]
`ccrba` is an algorithm. The other `robot.` method are also rather descriptive, like `robot.mass` instead of `robot.crba`. That's why I proposed the name `computeCentroidalMomentumMatrix`.

**Comment by jcarpent** [comment]
@jviereck I would just remove `compute` to mimic the current API of RobotWrapper ;)

**Comment by jcarpent** [comment]
Solved by #1111.

**Comment by jviereck** [comment]
Thanks for the quick fix @jcarpent !
